src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -95,6 +97,12 @@ const WorkspaceTabs = ({ namespace, name, workspace, activeTab, refresh }) => {
 }
 
 const WorkspaceContainer = ({ namespace, name, breadcrumbs, topBarContent, title, activeTab, showTabBar = true, refresh, refreshRuntimes, workspace, runtimes, persistentDisks, galaxyDataDisks, apps, refreshApps, children }) => {
+  const [deletingWorkspace, setDeletingWorkspace] = useState(false)
+  const [cloningWorkspace, setCloningWorkspace] = useState(false)
+  const [sharingWorkspace, setSharingWorkspace] = useState(false)
+  const isOwner = workspace && Utils.isOwner(workspace.accessLevel)
+  const canShare = workspace?.canShare
```

Comment:
I think this should be `workspace && workspace.canShare`. Right now this variable resolves to undefined if workspace is undefined (which is functionally identical since undefined is a falsey value), where as the new code ensures `canShare` always resolves to a boolean.

Reply:
Actually, `undefined && x` evaluates to `undefined`, due to short circuiting, not explicit boolean `false`. I'd say to have this as `!!workspace?.canShare`, as you need a `!!` to coerce to boolean either way.